.pre-commit-config.yaml

```
@@ -28,7 +29,7 @@ repos:
           # Do not use the configuration file but configure here so we can
           # enable more rules that are not mandatory in the configuration file.
           - "--select=B,E,F,I,ISC001,UP,W"  # TODO: select D
-          - "--ignore=E501"
+          - "--ignore=E501,B011,UP038"
```

```suggestion
          - "--ignore=E501"
```
Would it be OK with you to just fix these issues? It's only 4 of them anyway.